### DISCUSSION TITLE
Controller render example

### DISCUSSION THREAD

**Discussion by excid3** [josefarias/hotwire_combobox#65] [Ideas]
Sometimes it's handy to just render turbo streams from the controller without having a one-liner view. Would it be helpful to add this as an example to the docs?

```ruby
render turbo_stream: helpers.async_combobox_options @states
```

**Comment by josefarias** [comment]
Thanks @excid3. Yeah, I can see us adding this as a quick aside in the async combobox section of the docs. I'll keep this open until we do that 👍

**Comment by josefarias** [comment]
This is live now. Thanks again!

![Screengrab of the updated docs site showing an inline rendering example](https://github.com/josefarias/hotwire_combobox/assets/31393016/aba56311-c8da-44ba-b9aa-9916f12511a4)

**Reply by excid3** [reply]
Excellent! I'm planning to get back into development of my admin gem called Madmin, and Hotwire Combobox is probably going to be a core piece of it. 👍

**Reply by josefarias** [reply]
Love that! Happy to discuss any changes that would benefit your gem.
